src/renderer/dx/CustomTextRenderer.cpp

```diff
@@ -394,8 +400,7 @@ using namespace Microsoft::Console::Render;
                 case DWRITE_GLYPH_IMAGE_FORMATS_PNG:
                 case DWRITE_GLYPH_IMAGE_FORMATS_JPEG:
                 case DWRITE_GLYPH_IMAGE_FORMATS_TIFF:
-                case DWRITE_GLYPH_IMAGE_FORMATS_PREMULTIPLIED_B8G8R8A8:
-                {
+                case DWRITE_GLYPH_IMAGE_FORMATS_PREMULTIPLIED_B8G8R8A8: {

```

Comment:
These braces are breaking the formatting check:

```suggestion
                case DWRITE_GLYPH_IMAGE_FORMATS_PREMULTIPLIED_B8G8R8A8: 
                {
```

Reply:
Sorry about those, I don't why running Clang-Format made things against your formatting rules